src/client/setup.py

```diff
@@ -0,0 +1,57 @@
+#
```

Comment:
Does it need to be under src/client? Can we put it in src/client/pydaos and install it somewhere else as part of the rpm packaging / scons install?

Reply:
The reason this is here is so that you can install daos, checkout the src tree and then build the daos client _for your own python version_, and for that it does need to be here.

Again, this is something I'd like to address in future work, but I think having a directory src/client/python with the setup.py file and a daos directory would work better, then having the python module just called 'daos' rather than 'pydaos'.